client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss

```diff
@@ -159,8 +159,7 @@
 		}
 	}
 
-	&.videopress,
-	&.videopress-tv {
+	&.videopress {
```

Comment:
Is this videopress selector still needed? Will we render videopress components?

Reply:
I'm removing the `videopress` flow in a separate PR so there's less changes to review.